src/components/KFileUpload/KFileUpload.vue

```diff
@@ -0,0 +1,344 @@
+<template>
+  <div
+    class="k-file-upload w-100"
+  >
+    <KLabel
+      v-if="label && appearance === 'file'"
+      :for="customInputId"
+      v-bind="labelAttributes"
```

Comment:
If a label `help` attribute is bound, the tooltip icon doesn't have `cursor: pointer`